open-craft/edx-platform
# Add configuration variables for S3 region and signature type

This adds two extra configuration variables required to set both the S3 signature type, and the region for supporting ap-south-1 and other regions which require sigv4 for S3 access.

**JIRA tickets**: Implements BB-2950

**Dependencies**: None

**Screenshots**:

**Merge deadline**: "ASAP"

**Testing instructions**:

1. Export course on patched instance
2. Ensure that download link for exported course works


**Reviewers**
